Upstream: No
Reason: Software crashes somewhere in librsvg/cairo/pixman while loading
icons. This patch simply removes the icon loading for now, meaning we wont
have icons in Software, but at least it wont crash. This is just a workaround
for now, so Software doesn't crash for our users anymore.
diff --git a/plugins/core/meson.build b/plugins/core/meson.build
index 39630d1..5f6dcc2 100644
--- a/plugins/core/meson.build
+++ b/plugins/core/meson.build
@@ -80,23 +80,6 @@ shared_module(
   ]
 )
 
-
-shared_module(
-  'gs_plugin_icons',
-  sources : 'gs-plugin-icons.c',
-  include_directories : [
-    include_directories('../..'),
-    include_directories('../../lib'),
-  ],
-  install : true,
-  install_dir: plugin_dir,
-  c_args : cargs,
-  dependencies : plugin_libs,
-  link_with : [
-    libgnomesoftware
-  ]
-)
-
 shared_module(
   'gs_plugin_appstream',
   sources : [
